### DISCUSSION TITLE
Welcome to AppiumTestDistribution Discussions!

### DISCUSSION THREAD

**Discussion by saikrishna321** [AppiumTestDistribution/AppiumTestDistribution#802] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sai417** [comment]
I have given capabilities to iOS device like this, but showing no devices connected!
"iOS": {
		"platformName": "iOS",
		"deviceName": "SaiPad",
		"bundleID": "com.XYZ",
        "xcodeSigningId": "iPhone Developer",
        "platformVersion": "13.6.1",
        "udid": "XXXX-2E",
		"wdaLaunchTimeout": 80000,
		"wdaConnectionTimeout": 80000,
		"preventWDAAttachments": true,
		"clearSystemFiles": true,
		"shouldUseSingletonTestManager": false,
		"simpleIsVisibleCheck": true,
		"noReset": true,
        "newWDA" : false,
		"maxTypingFrequency": 10
	},
	"hostMachines": [
		{
			"machineIP": "127.0.0.1",
			"appiumServerPath": "/usr/local/lib/node_modules/appium/build/lib/main.js"
		}
	]

Can you please help me, did I miss anything!
